doc/api/cli.md

```diff
@@ -1196,6 +1196,16 @@ added: v19.0.0
 
 Disable exposition of [CustomEvent Web API][] on the global scope.
 
+### `--no-experimental-global-navigator`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
```

Comment:
```suggestion
> Stability: 1.2 - Release candidate
```

Reply:
It's not clear at all that we want to ever ship this flag as stable, I don't think Release candidate is the correct stage, I'd keep it as Experimental for now.